src/m365/spo/commands/site/site-classic-set.ts

```diff
@@ -191,6 +193,30 @@ class SpoSiteClassicSetCommand extends SpoCommand {
             });
         });
       })
+      .then((): Promise<FormDigestInfo> => {
+        return this.ensureFormDigest(this.spoAdminUrl as string, logger, this.context, this.debug);
+      })
+      .then((res: FormDigestInfo): Promise<void> => {
+        this.context = res;
+        
+        if (!args.options.description) {
+          return Promise.resolve(undefined as any);
+        }
+
+        const requestOptions: any = {
+          url: `${args.options.url}/_api/web`,
+          headers: {
+            'IF-MATCH': '*',
+            'Accept': 'application/json;odata=verbose',
+            'content-type': 'application/json;odata=verbose',
+            'X-RequestDigest': (this.context as FormDigestInfo).FormDigestValue,
+            'X-HTTP-Method': 'MERGE'
+          },
+          data: `{ '__metadata': { 'type': 'SP.Web' }, 'Description': '${args.options.description}' }`
```

Comment:
Rather than manually building a JSON string, we should let the CLI do this, by setting `data: { Description: args.options.description }, json: true`